File: backend/types/thing.go
package types

type LostThing struct {
	Id                  string
	PublicationDatetime string
	Name                string
	UserEmail           string
	UserMessage         string
	Verified            int
	Found               int
	AdvertisementOwner  string
}

type FoundThing struct {
	Id                  string
	PublicationDatetime string
	Name                string
	Location            string
	UserMessage         string
	Verified            int
	Found               int
	AdvertisementOwner  string
}

File: frontend/src/types/thing.ts
import { ThingType } from '../utils/consts';

export interface Thing {
  Id: string;
  Type: ThingType;
  PublicationDatetime: string;
  Name: string;
  Photo: string;
  UserMessage: string;
  Verified: string;
  Found: string;
  NoticeOwner: string;
}
